src/backend/utils/adt/agtype.c

```diff
@@ -2575,7 +2575,7 @@ Datum agtype_to_bool(PG_FUNCTION_ARGS)
     agtype_value agtv;
 
     if (!agtype_extract_scalar(&agtype_in->root, &agtv) ||
-        agtv.type != AGTV_BOOL)
+        (agtv.type != AGTV_BOOL && agtv.type != AGTV_INTEGER))
```

Comment:
All ifs need to have {} even for 1 line of code. If you are modifying old code that doesn't follow that standard, please update it. No need to update everything, just the contents of the function.